build/jQueryShim.js

```diff
@@ -88,6 +88,9 @@ var ajax = function ajax(options) {
 
   request.withCredentials = options.xhrFields.withCredentials;
   var cacheBuster = "&_=" + new Date().getTime();
+  if (options.url.indexOf("?") === -1) {
```

Comment:
Is this logic really correct?

If the url is just `http://www.domain.com/url` it will become  `http://www.domain.com/url?&_=123123`

Reply:
It should be `http://www.domain.com/url?_=123123`